src/rtc/rtc_s1_single_job.py

```diff
@@ -1426,9 +1429,14 @@ def run_single_job(cfg: RunConfig):
 
                 logger.info('    computing layover shadow mask for'
                             f' {burst_id}')
+
+                radar_grid_layover_shadow_mask = radar_grid.multilook(
+                    STATIC_LAYERS_LAYOVER_SHADOW_MASK_MULTILOOK_FACTOR,
+                    STATIC_LAYERS_LAYOVER_SHADOW_MASK_MULTILOOK_FACTOR)
```

Comment:
So it looks like you are down-sampling the radargrid for the layover shadow mask using multilook. When we started to work on the layover shadow mask, I think you have mentioned that we need to run it in full radargrid resolution. No?

Reply:
Yes, we still need that for RTC-S1 products. It's just for static layers that we can reduce the spacing of the radar grid and I'm only doing that because it's running so slow =/

Reply:
I see. I've quickly tested the static layer mode of the workflow, and in fact `rtc_s1_single_job.py` was about twice faster than `rtc_s1.py` (5m23s vs. 11m24s respectively). One of the reasons I can think about is that we limit the number of threads to `1` for each child processes in parallelized workflow.
I wonder if you can also see faster processing time from `rtc_s1_single_job.py` for static layer processing. If yes, what do you think about letting the users to use the that script for static layer processing?

Reply:
Thank you for testing it, @seongsujeong . Yes, in fact I mostly use `rtc_s1_single_job.py` for testing. 

I'd say that this static layers processing is not really useful for most (if not all) users. Notice that users can still generate any "secondary layer" by changing the corresponding flags in the runconfig. My biggest concern with the static layers processing is the SDS processing and in that case, it's probably better to use `rtc_s1.py` script.